src/riak_cs_bw_limitter.erl

```diff
@@ -0,0 +1,235 @@
+%% ---------------------------------------------------------------------
+%%
+%% Copyright (c) 2007-2015 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% ---------------------------------------------------------------------
+
+%% @doc Simple node-level rate limit implementation, per user
+%% This is experimental feature.
+%%
+%% riak_cs_bw_limitter help
```

Comment:
![Limitter](https://s3.amazonaws.com/uploads.hipchat.com/20796/154834/kZS2mRllxurfwMZ/Screenshot_from_2015-04-13%2009%3A49%3A10.png) 

and

![Limiter](https://s3.amazonaws.com/uploads.hipchat.com/20796/154834/RNR6xHDcPxmv7te/Screenshot_from_2015-04-13%2009%3A49%3A38.png)

I'll change the whole words to limiter, and module name will be `riak_cs_simple_bw_limiter`, aligning to the quota one.